### ISSUE TITLE
can't deploy with jets

### ISSUE THREAD

**Issue by yu-coder** [tongueroo/jets#363]
Hi, I created rails API and was going to deploy AWS Lambda API using Jets.
But I can't deploy using jets deploy command.
I could deploy 2 weeks ago with the older version. It seems there should be an issue on the latest version
This is error detail.
```
jets deploy
=> Rails app detected: Enabling Jets Afterburner to deploy to AWS Lambda.
Deploying to Lambda my-project-dev environment...
Building CloudFormation templates.
Deploying CloudFormation stack with jets app!
08:22:14AM CREATE_IN_PROGRESS AWS::CloudFormation::Stack my-project-dev User Initiated
08:22:16AM CREATE_IN_PROGRESS AWS::S3::Bucket S3Bucket 
08:22:16AM CREATE_IN_PROGRESS AWS::S3::Bucket S3Bucket Resource creation Initiated
08:22:37AM CREATE_COMPLETE AWS::S3::Bucket S3Bucket 
08:22:38AM CREATE_COMPLETE AWS::CloudFormation::Stack my-project-dev 
Stack success status: CREATE_COMPLETE
Time took for stack deployment: 25s.
=> Compling assets in current project directory
=> Copying current project directory to temporary build area: /tmp/jets/my-project/stage/code
Traceback (most recent call last):
	28: from /Users/admin/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `<main>'
	27: from /Users/admin/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `eval'
	26: from /Users/admin/.rvm/gems/ruby-2.5.3/bin/jets:23:in `<main>'
	25: from /Users/admin/.rvm/gems/ruby-2.5.3/bin/jets:23:in `load'
	24: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/exe/jets:14:in `<top (required)>'
	23: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/cli.rb:5:in `start'
	22: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/cli.rb:28:in `start'
	21: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/commands/base.rb:27:in `perform'
	20: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/commands/base.rb:38:in `dispatch'
	19: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	18: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	17: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	16: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/commands/main.rb:21:in `deploy'
	15: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/commands/deploy.rb:33:in `run'
	14: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/commands/deploy.rb:85:in `build_code'
	13: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/commands/build.rb:27:in `build_code'
	12: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/builders/code_builder.rb:37:in `build'
	11: from /Users/admin/.rvm/gems/ruby-2.5.3/gems/jets-2.1.5/lib/jets/builders/code_builder.rb:377:in `copy_ruby_version_file'
	10: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:392:in `cp_r'
	 9: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1461:in `fu_each_src_dest'
	 8: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1477:in `fu_each_src_dest0'
	 7: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1463:in `block in fu_each_src_dest'
	 6: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:393:in `block in cp_r'
	 5: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:415:in `copy_entry'
	 4: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1390:in `wrap_traverse'
	 3: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:418:in `block in copy_entry'
	 2: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1256:in `copy'
	 1: from /Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1227:in `lstat'
/Users/admin/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1227:in `stat': No such file or directory @ rb_file_s_stat - /tmp/jets/turbo-wrapper-project/.ruby-version (Errno::ENOENT)

```

**Comment by yu-coder** [comment]
Hi @tongueroo could you please help to fix?

**Comment by tongueroo** [comment]
Fixed in #366. 

Had enough info here to fix the edge case bug. In the future, please make sure to fill github issue template. Thanks!

**Comment by yu-coder** [comment]
Thank you for your help!
